rfcs/react-components/convergence/semantic-shared-colors.md

```diff
@@ -0,0 +1,195 @@
+| `colorPaletteRedBackground1`        | status                |
+| `colorPaletteRedBackground2`        | status + persona      |
+| `colorPaletteRedBackground3`        | status                |
+| `colorPaletteRedForeground1`        | status                |
+| `colorPaletteRedForeground2`        | status + persona      |
+| `colorPaletteRedForeground3`        | status                |
+| `colorPaletteRedForegroundInverted` | Red, Green and Yellow |
+| `colorPaletteRedBorderActive`       | status + persona      |
+| `colorPaletteRedBorder1`            | status                |
+| `colorPaletteRedBorder2`            | status                |
+
+This results in 150 (28 _ 3 + 7 _ 9 + 3) color tokens.
+
+Adding new tokens comes at a [runtime performance cost](https://github.com/microsoft/fluentui/blob/master/rfcs/react-components/convergence/theme-shape.md).
+
+## Problem statement
+
+There is no semantic meaning in the current shared color tokens. The intended use case is communicated only by a design spec. See [#26168](https://github.com/microsoft/fluentui/issues/26168), [#25720](https://github.com/microsoft/fluentui/issues/25720).
+Designers plan to update the color values for the status shared colors - that might seem just like a hex value change, but that results in a color name change: for example **red** will be replaced by **cranberry**.
+
+### Token change request from design
+
+The current requirement from design is to keep **presence** and **persona** token names and introduce semantic names for **status** tokens. This is the ideal end state:
+
+| Current name | Current meaning       | Proposed change                                                                                                                     |
+| ------------ | --------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| `red`        | danger, busy, persona | Introduce semantic `danger`, mapped to `cranberry` instead of `red`.<br/>Keep `red`, it is still used for **busy** and **persona**. |
+| `green`      | success               | Replace by semantic `success`.                                                                                                      |
+| `darkOrange` | severe                | **severe** should not be used anywhere. Keep `darkOrange` (temporarily) for Teams compatibility.                                    |
+| `yellow`     | warning               | Replace by semantic `warning`, mapping changed from `yellow` to `orange`                                                            |
+| `berry`      | OOF                   | No change                                                                                                                           |
+| `lightGreen` | available             | No change                                                                                                                           |
```

Comment:
It doesn't look consistent that some tokens are named after colors e.g. `lightGreen` and some are after statuses, e.g. `success`. Maybe in future all tokens will use semantic names but what if these tokens will be needed not only for statuses

Reply:
Agree here. Let's take the current usages of `red` as an example (and for now ignore the fact that we are changing the danger from red to cranberry).

`red` is used for:
1. danger
2. busy
3. persona

Now we will introduce `danger` _status_ semantic color. But we cannot deprecate the `red` because that is still used for busy and persona.

Eventually we might decide to introduce _presence_ semantic colors and add `busy`. After that we still need the `red` for persona so we cannot remove it, but can deprecate the slots used only for _presence_.

Also, this covers only the usages in the core library, but not partners' codebases. Design provides partners with guidelines but we cannot be 100% sure about what they really do.